web-api/src/persistence/postgres/cases/getCaseMetadataByDocketNumber.ts

```diff
@@ -26,18 +25,13 @@ export const getCaseMetadataByDocketNumber = async ({
   );
 
   return dbCase
-    ? {
-        ...transformNullToUndefined(rawCaseEntity(dbCase)),
-        petitioners:
-          (dbCase.petitioners as RawPetitioner[]).map(p => {
-            if (!p) {
-              return;
-            }
-            return new Petitioner({
-              ...transformNullToUndefined(p),
-              state: p.state || null, // this needs to be null
-            }).toRawObject();
-          }) || [],
-      }
+    ? fromKyselyCase({
+        ...dbCase,
+        petitioners: dbCase.petitioners as TPetitioner[], // This is a hack because our typing for Petitioners is yucky
+        docketNumberWithSuffix: Case.getDocketNumberWithSuffix({
+          docketNumber: dbCase.docketNumber,
+          docketNumberSuffix: dbCase.docketNumberSuffix,
+        }),
```

Comment:
We could put this in the mapper. Why didn't I? I don't have super strong feelings, but the way the mapper currently works is "here is some database data and random stuff please map it." `docketNumberWithSuffix` is not database data.

Reply:
Won't die on this hill, but counterpoint: `docketNumberWithSuffix` is required in 90% of usecases, shouldn't we make it difficult to forget to add in new functions?

Reply:
I don't know, I keep going back and forth on this